tailwind.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
       "codify-light-purple": "#B57EE8",
       "link-blue": "#4A90E2",
       "subheader-gray": "#808080",
+      "footer-dark-purple": "#29274C"
```

Comment:
If you think we should only use this color in the footer then leaving it named footer-dark-purple is fine, but if we should consider using it elsewhere then I would rename to to a more general name.